liquidctl/driver/hydro_platinum.py

```diff
@@ -126,6 +127,8 @@ class HydroPlatinum(UsbHidDriver):
             {'fan_count': 2, 'fan_leds': 0}),
         (0x1b1c, 0x0c22, 'Corsair Hydro H150i Pro XT',
             {'fan_count': 3, 'fan_leds': 0}),
+        (0x1b1c, 0x0c35, 'Corsair iCUE H100i RGB ELITE',
+            {'fan_count': 2, 'fan_leds': 0, 'has_pump':True}),
```

Comment:
`has_pump` also does nothing here, and use "our" name for the device:

```suggestion
        (0x1b1c, 0x0c35, 'Corsair iCUE H100i Elite RGB',
            {'fan_count': 2, 'fan_leds': 0}),
```